README.md

```diff
@@ -1057,8 +1028,7 @@ documentation](https://docs.rubocop.org/rubocop/configuration.html#inheriting-co
   /(?:first|second)/
   ~~~
 
-* Don't use the cryptic Perl-legacy variables denoting last regexp group matches
-  (`$1`, `$2`, etc). Use `Regexp#match` instead.
+* Prefer `Regexp#match` over Perl-legacy variables to capture group matches.
```

Comment:
Keeping

> (`$1`, `$2`, etc)

would make it clearer what this refers to for people who haven't used Perl.

Reply:
I used the code sample to exemplify `$1`. We should always prefer to clarify a rule with a code sample with necessary (instead of using extra sentences or parentheses).

Reply:
Like that line of thinking!